frontend/src/layout/navigation/TopBar/Announcement.tsx

```diff
@@ -1,20 +1,30 @@
-import React, { Fragment } from 'react'
+import React from 'react'
 import ReactMarkdown from 'react-markdown'
 import clsx from 'clsx'
 import { CloseOutlined } from '@ant-design/icons'
 import { MOCK_NODE_PROCESS } from 'lib/constants'
 import { announcementLogic, AnnouncementType } from '~/layout/navigation/TopBar/announcementLogic'
 import { useActions, useValues } from 'kea'
 import { GroupsIntroductionBanner } from 'lib/introductions/GroupsIntroductionBanner'
+import { preflightLogic } from 'scenes/PreflightCheck/logic'
 
 window.process = MOCK_NODE_PROCESS
 
 export function Announcement(): JSX.Element | null {
-    const { shownAnnouncementType, cloudAnnouncement } = useValues(announcementLogic)
+    const { shownAnnouncementType, cloudAnnouncement, closable } = useValues(announcementLogic)
+    const { preflight } = useValues(preflightLogic)
     const { hideAnnouncement } = useActions(announcementLogic)
 
-    let message = <Fragment />
-    if (shownAnnouncementType === AnnouncementType.CloudFlag && cloudAnnouncement) {
+    let message: JSX.Element | undefined
+    if (preflight?.demo) {
+        message = (
+            <b>
+                Welcome to PostHog's demo environment. To level up,{' '}
+                <a href="https://posthog.com/signup/self-host">easily deploy PostHog yourself</a> or{' '}
+                <a href="https://app.posthog.com/signup">sign up for a free PostHog Cloud account</a>.
```

Comment:
Nit: https://posthog.com/signup covers both self-hosting and cloud and we should likely just redirect them there, and that page can evolve and give all the guidance. "easily deploy" might mislead folks to thinking it's easier to self-host compared to cloud, which isn't true. Related Q: do we want that page to highlight the option to check out the demo too?

Reply:
Good point, simplified to a single link to that "Get started" page with copy "deploy your own PostHog instance or sign up for PostHog Cloud". I think if we go forward with the demo env, it should be somewhere on the website, but the homepage seems like the clearest place to expose it, so I'll stay with that for now (https://github.com/PostHog/posthog.com/pull/2661).